src/frontend/web/src/ratingpage/RatingCard.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
I have an idea.
Could this card be good also for the match page? @mryuv1 
If so, please export it to `components`, make it generic, and @mryuv1 add it to your page :)
We shouldn't have double work.

Reply:
In the match page, we need to contain more information, therefore if the card is flexible it can fit. What do you think? @Lotemn102

Reply:
Great idea! i'll make it generic (it's height and width as well, if that's what you mean by flexible @mryuv1), and add extra optional lines for more info.